app/components/school_information/your_schools/participants_school_search_component.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="govuk-grid-row">
+  <%= component_form_with model: @school_search_form, url: lead_providers_your_schools_path, method: :post do |f| %>
```

Comment:
How does this differ from `form_with`?

Reply:
Is it just that we need it because it's in a component?

Reply:
for components - you would need to pass gov form builder in params - I've created a wrapper so that you wouldn't have to do this

Reply:
Came as quite a surprise